bson/mgocompat/doc.go

```diff
@@ -1,14 +1,14 @@
 // Package mgocompat provides MgoRegistry, a BSON registry compatible with globalsign/mgo's BSON,
 // with some remaining differences. It also provides MgoRegistryRespectNilValues for compatibility
-// with mgo's BSON with RespectNilValues set to true. See the bsoncodec docs for more details on
-// registries.
+// with mgo's BSON with RespectNilValues set to true. A registry can be configured on a
+// mongo.Client with the SetRegistry option. See the bsoncodec docs for more details on registries.
```

Comment:
Any further updates to this can be made in scope of https://jira.mongodb.org/browse/GODRIVER-1361